File: Java/creational_patterns/src/Builder/SecondHouse.java
package Builder;

public class SecondHouse extends Builder {
    public SecondHouse() {
        house = new House();
    }

    @Override
    public House builHouse() {
        house.buildBase();
        house.buildWalls();
        house.buildRoof();
        return house;
    }
}

File: C#/CreationalPatterns/CreationalPatterns.Builder/SecondHouse.cs
namespace CreationalPatterns.Builder
{
    internal class SecondHouse:Builder
    {
        public SecondHouse()
        {
            house = new House();
        }

        public override House BuildHouse()
        {
            house.BuildBase();
            house.BuildWalls();
            house.BuildRoof();
            return house;
        }
    }
}
